app/front/scripts/directives/export-control.js

```diff
@@ -0,0 +1,88 @@
+;(function(angular) {
+
+  var components = require('components');
+
+  var app = angular.module('Application');
+
+  function processCell(rows, rowIndex, colspan, rowspan, text) {
+    var colIndex = 0;
+    while (rows[rowIndex][colIndex] !== undefined) {
```

Comment:
No, we cant, because there may be table like:
```
+---+---+---+---+
|   | 2 |   | 4 |  
+ 1 +---+ 3 +---+
|   | 5 |   | 6 |
+---+---+---+---+
```
When processing first row, we'll get arrays like:
```
[1, 2, 3, 4] (length == 4)
["", undefined, ""] (length == 3)
```
So when processing next row, we need to place 5 instead of first `undefined`, and append 6.